base/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM continuumio/miniconda3:4.8.2
+FROM condaforge/miniforge3:latest
```

Comment:
I think using a concrete version that works should be preferred than `latest`?

Reply:
Yeah I can pick a specific version, I went with latest since that had arm support.